frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  IconButton,
+  MenuList,
+  Paper,
+  Popper,
+} from "@material-ui/core";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+
+const StyledNotificationsIcon = styled(IconButton)`
+  color: #ffffff;
+  margin-right: 1rem;
+  border-radius: 8px;
+  padding: 8px;
+  &:hover {
+    background: #2d3db4;
+  }
+  &:active {
+    background: #2938a5;
+  }
+`;
+
+const Notifications: React.FC = () => {
```

Comment:
We might want to provide an interface to pass in notifications here to render in the popper

Reply:
do you mean something as simple as an interface with just `text: string` for now?

Reply:
Resolving in lieu of https://github.com/lyft/clutch/pull/696#discussion_r534655596